### ISSUE TITLE
pretrained model can't open

### ISSUE THREAD

**Issue by mint0126** [calmevtime/DCTNet#46]
Could you please provide the pretrained model again? The model downloaded from the original link shows that the compression is damaged and cannot be opened. plz help me

**Comment by mint0126** [comment]
The classification pretrained model can't be opend.

**Comment by hwei-hw** [comment]
you can find the classification pre-trained model at [here](https://drive.google.com/drive/folders/1eC9xBexK4aKoNzPRiLU2sYu_UyYm3TRU), but only including the model of setting in static_24 and static_64, without the model of dynamic setting (input channel=192). 
I need the version of the dynamic model (input channel=192). I would appreciate it very much if someone could provide it!

**Comment by liliiscaiji** [comment]
> you can find the classification pre-trained model at [here](https://drive.google.com/drive/folders/1eC9xBexK4aKoNzPRiLU2sYu_UyYm3TRU), but only including the model of setting in static_24 and static_64, without the model of dynamic setting (input channel=192). I need the version of the dynamic model (input channel=192). I would appreciate it very much if someone could provide it!

Hello, may I ask if your pre-training weights can be downloaded completely? All the pre-training weight compression files I downloaded are corrupted！

**Comment by hwei-hw** [comment]
Yes, I can download these pre-training models completely and also can load these weights correctly into the model. Maybe your network is unstable.

**Comment by guascy666** [comment]
> Yes, I can download these pre-training models completely and also can load these weights correctly into the model. Maybe your network is unstable.

hello can you provide the downloaded pth file which has been decompressed？i need the resnet50 static64 ，i cannot decompressed the pretrained model
